### PR TITLE
Add a bedspace to a property

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

* A user can add a bedspace

## Screenshots of UI changes

localhost_3000_temporary-accommodation_properties_98e42f78-b39a-4336-905d-1f97fd618fe7_bedspaces_new

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[Add a bedspace to a property](https://trello.com/c/XnnFmXfU/486-add-a-bedspace-to-a-property)

File: server/services/bedspaceService.test.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import newRoomFactory from '../testutils/factories/newRoom'
+import RoomClient from '../data/roomClient'
+import BedspaceService from './bedspaceService'
+import ReferenceDataClient from '../data/referenceDataClient'
+import characteristic from '../testutils/factories/characteristic'
+
+jest.mock('../data/roomClient')
+jest.mock('../data/referenceDataClient')
+
+describe('BedsapceService', () => {
```
Comment (richpjames) [inline server/services/bedspaceService.test.ts]: typo here

File: integration_tests/tests/temporary-accommodation/manage/bedspace.cy.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+context('Bedspace', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+  })
+
+  it('allows me to create a bedspace', () => {
+    // Given I am signed in
```
Comment (richpjames) [inline integration_tests/tests/temporary-accommodation/manage/bedspace.cy.ts:17]: Completely up to you but if we were starting again I think we would use the pattern established in `manage.feature` of using Gherkin syntax as opposed to just commenting it but it takes more setup/boilerplate

**Review by richpjames — DISMISSED** [review]
LGTM

**Review by richpjames — APPROVED** [review]

**Pull request merged**